modules/layers/src/text-layer/utils.js

```diff
@@ -69,42 +73,110 @@ export function buildMapping({
   };
 }
 
-export function transformRow(row, iconMapping, lineHeight) {
+export function autoWrapping({
+  string,
+  lineHeight,
+  width,
+  wordBreak,
+  fontSize,
+  fontFamily,
+  fontWeight,
+  textAlign
+}) {
+  const characters = Array.from(string);
+  const textDiv = document.createElement('div');
```

Comment:
We should not be creating/removing DOM objects for each string. This can be reused.